### PR TITLE
VIH-7205 video web

### PR DESCRIPTION
**Pull request by stravze**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/vh-video-web/values.yaml [hunk 1]
```diff
@@ -44,20 +42,21 @@ java:
...
+    VHSERVICES__USERAPIURL: https://vh-user-api.{{ .Values.global.environment }}.platform.hmcts.net/
     VHSERVICES__VIDEOAPIESOURCEID: https://vh-video-api.{{ .Values.global.environment }}.platform.hmcts.net
-    VHSERVICES__VIDEOAPIURL: https://vh-video-api.{{ .Values.global.environment }}.platform.hmcts.net
-    VHSERVICES__VIDEOWEBAPIURL: https://vh-video-web.{{ .Values.global.environment }}.platform.hmcts.net
+    VHSERVICES__VIDEOAPIURL: https://vh-video-api.{{ .Values.global.environment }}.platform.hmcts.net/
+    VHSERVICES__VIDEOWEBAPIURL: https://vh-video-web.{{ .Values.global.environment }}.platform.hmcts.net/
     VHSERVICES__VIDEOWEBRESOURCEID: https://vh-video-web.{{ .Values.global.environment }}.platform.hmcts.net
-    VHSERVICES__VIDEOURL: https://vh-video-web.{{ .Values.global.environment }}.platform.hmcts.net
-    VideoAppUrl: https://vh-video-web.{{ .Values.global.environment }}.hearings.reform.hmcts.net
+    VHSERVICES__VIDEOWEBURL: https://vh-video-web.{{ .Values.global.environment }}.platform.hmcts.net
```
Comment (levibutler) [inline charts/vh-video-web/values.yaml:55]: thats this variable for?

Comment (levibutler) [reply charts/vh-video-web/values.yaml:55]: Im only going from diff, but maybe should have been `VHSERVICES__VIDEOURL` ?

File: charts/vh-video-web/values.yaml [hunk 2]
```diff
@@ -44,20 +42,21 @@ java:
     MSDEPLOY_RENAME_LOCKED_FILES: 1
     TESTING__TESTUSERNAMESTEM: "@hearings.reform.hmcts.net"
     VHSERVICES__BOOKINGSAPIRESOURCEID: https://vh-bookings-api.{{ .Values.global.environment }}.platform.hmcts.net
-    VHSERVICES__BOOKINGSAPIURL: https://vh-bookings-api.{{ .Values.global.environment }}.platform.hmcts.net
+    VHSERVICES__BOOKINGSAPIURL: https://vh-bookings-api.{{ .Values.global.environment }}.platform.hmcts.net/
+    VHSERVICES__TESTAPIRESOURCEURL: https://vh-test-api.{{ .Values.global.environment }}.platform.hmcts.net
```
Comment (levibutler) [inline charts/vh-video-web/values.yaml:46]: this one new?

**Comment by levibutler** [comment]
Think i might need to check what settings we need? maybe diff isnt a reliable source if its wrong :P

**Review by cts-code — APPROVED** [review]
Approved

**Pull request merged**